scripts/tfenv-install-terraform.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e
+
+
+if [ -f "~/.tfenv" ]; then
+  rm -rf ~/.tfenv
+fi
+
+git clone https://github.com/tfutils/tfenv.git ~/.tfenv
```

Comment:
always use a released version
```suggestion
git clone -b v2.2.2 --single-branch https://github.com/tfutils/tfenv.git ~/.tfenv
```

Reply:
updated